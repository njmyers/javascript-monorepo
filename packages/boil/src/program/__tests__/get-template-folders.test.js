import getTemplateFolders from '../get-template-folders';

describe('it works', () => {
  test('it is a function ', () => {
    expect(typeof getTemplateFolders()).toBe('function');
  });

  test('it runs without crashing', () => {
    expect(getTemplateFolders()).toMatchSnapshot();
  });
});
