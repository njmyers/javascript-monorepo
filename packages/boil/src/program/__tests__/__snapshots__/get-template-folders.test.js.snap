// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`program/getTemplateFolders it runs without crashing 1`] = `
Array [
  "/home/developer/Projects/boil/README.md",
  "/home/developer/Projects/boil/babel.config.js",
  "/home/developer/Projects/boil/build",
  "/home/developer/Projects/boil/jest.config.js",
  "/home/developer/Projects/boil/node_modules",
  "/home/developer/Projects/boil/package.json",
  "/home/developer/Projects/boil/rollup.config.js",
  "/home/developer/Projects/boil/scripts",
  "/home/developer/Projects/boil/src",
  "/home/developer/Projects/boil/templates",
  "/home/developer/Projects/boil/yarn.lock",
]
`;
